cdap-docs/developers-manual/source/building-blocks/applications.rst

```diff
@@ -51,6 +51,38 @@ name.
 Names used for streams and datasets need to be unique across the CDAP instance, while
 names used for programs and services need to be unique only to the application.
 
+Application can also use a Configuration class to get a configuration during deployment time of the Application.
```

Comment:
need some more detail.  For example, how is the configuration specified? Maybe link to the rest api section for more detail.